src/swarm/neo/request/RequestEventDispatcher.d

```diff
@@ -281,24 +286,25 @@ public struct RequestEventDispatcher
     /// List of signals which occurred while the request-on-conn fiber was
     /// running. (Signals which occur when the fiber is suspended are passed to
     /// the fiber resume method, waking it up.)
-    private ubyte[] queued_signals;
+    private VoidBufferAsArrayOf!(ubyte) queued_signals;
 
     /***************************************************************************
 
-        Resets this instance to its initial state. This method should be called
-        when retrieving an instance from a pool or free list.
+        Initialises the RequestEventDispatcher setting it's initial state.
+
+        Params:
+            getVoidArray = void array acquirer.
```

Comment:
Maybe add "This instance will call this delegate multiple times, to acquire the buffers it needs internally. The request's acquired resources must not be relinquished during the lifetime of this instance.".